desc/coils.py

```diff
@@ -267,6 +269,15 @@ def to_SplineXYZ(self, knots=None, grid=None, method="cubic", name=""):
             - `'catmull-rom'`: C1 cubic centripetal "tension" splines
         name : str
             name for this coil
+        break_indices : ndarray
```

Comment:
can you add something about hte default behavior, like "If not given, then the usual periodic spline is assumed" or at the start say "If not None, then the spline will be a piecewise set of splines..." or something so the default behavior is known, and put that the break_indices can be None `"break_indices : ndarray or None"` or something

Reply:
Okay let me know if what I have now is good